cdap-docs/examples-manual/source/examples/spark-page-rank.rst

```diff
@@ -23,8 +22,13 @@ to the *backlinkURLStream*, which stores the URL pair event in its entirety.
 After these events are streamed, they are taken up by the *SparkPageRankProgram*, which
 goes through the entries, calculates page rank and tabulates results in an ObjectStore dataset, *ranks*.
 
+A MapReduce job uses the output of the Spark program from *ranks* dataset and computes the total number of pages for every unique page rank and tabulates its results in another ObjectStore dataset, *rankscount*.
+
+The *PageRankWorkflow* ties the Spark and MapReduce to run together sequentially in this application.
+
 Once the application completes, you can query the *ranks* dataset by using the ``rank`` endpoint of the *RanksService*.
-It will send back a string result with page rank based on the ``url`` query parameter.
+It will send back a string result with page rank based on the ``url`` query parameter. You can also query the
+*rankscount* dataset by using ``total`` endpoint of the *TotalPagesPR*. It will send the total number of pages which has the page rank queried for as a string.
```

Comment:
with the query supplied as string makes it sound like that the query was a string whereas I want to say that the result which will be sent will be a string. I am doing this:
```
It will send the total number of pages for the queried page rank as a string.
```
Let me know if its fine.